toolchains/linux.toolchain.cmake

```diff
@@ -0,0 +1,57 @@
+################################################################################
+#
+# T:N.U.N. Apple OS X CMake tool chain file.
+#
+# Copyright (c) 2016. Domagoj Saric. All rights reserved.
+#
+################################################################################
+
+# MACOSX_RPATH
+
+set( CPACK_SYSTEM_NAME "Linux" )
+
+set( LINUX true )
+
+set( TNUN_os_suffix Linux )
+
+if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
+    include( "${CMAKE_CURRENT_LIST_DIR}/clang.cmake" )
+else()
+    include( "${CMAKE_CURRENT_LIST_DIR}/gcc_compatibles.cmake" )
+endif()
+
+option( TNUN_STATIC_LIBC "Link all binaries statically with libc and libc++" false )
```

Comment:
preferably this would be a more generic option (e.g. also often used on windows)...but ok we can add this in a separate change..